conseil-common/src/main/scala/tech/cryptonomic/conseil/common/mantis/Utils.scala

```diff
@@ -0,0 +1,30 @@
+package tech.cryptonomic.conseil.common.mantis
+
+import scala.util.Try
+
+import scorex.crypto.hash.Keccak256
+
+/**
+  * Utils methods required by the Mantis integration.
+  */
+object Utils {
```

Comment:
Why not placed in `conseil-common/src/main/scala/tech/cryptonomic/conseil/common/util/` as  `HexUtils.scala`?